cas/service.c

```diff
@@ -23,9 +23,9 @@
 #define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_CAS
```

Comment:
For CAS: addressed in https://github.com/Seagate/cortx-motr/pull/802/commits/c75464d7d2ce0553ee63b8fa00745944e6dee137#diff-9b8c53be15e4d5294c848cefa89c9bde1e2e0fbc716e4e30ada41aab31d530d9

For rpc/link: addressed in https://github.com/Seagate/cortx-motr/pull/802/commits/c57e49a7acbc80feab03b94a9e42c3514f7129bb

For coroutines: it will be addressed separately (see my previous comment).